### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by hendridg**
# Learning objectives

- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code asynchronous.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,24 +1,79 @@
...
+const urlNewGame = `
+  https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/`;
+const nameGame = { name: "Hendrid's cool name" };
 
 if (localStorage.savedPlayers) {
   objPlayers.players = JSON.parse(localStorage.getItem('savedPlayers'));
 }
 
-form.addEventListener('submit', (e) => {
+async function keyNewGameAPIs() {
```
Comment (ZeenatLawal) [inline src/index.js]: - [ ] Kindly make use of arrow functions to follow best practices.

File: src/index.js [hunk 2]
```diff
@@ -1,24 +1,79 @@
...
+      'Content-Type': 'application/json',
+    },
+  });
+  const data = await response.json();
+  return data.result.slice(14, 34);
+}
+
+const keyGame = keyNewGameAPIs();
+
+async function getAPIs() {
```
Comment (ZeenatLawal) [inline src/index.js]: - [ ] Kindly make use of arrow functions to follow best practices.

File: src/index.js [hunk 3]
```diff
@@ -1,24 +1,79 @@
...
+  const response = await fetch(`
+    https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${keyGame}/scores`);
+  const data = await response.json();
+  objPlayers.players = data.result;
+  objPlayers.displayPlayers();
+  return data.result;
+}
+
+async function postPlayer(newPlayer) {
```
Comment (ZeenatLawal) [inline src/index.js]: - [ ] Kindly make use of arrow functions to follow best practices.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @hendridg, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
## STATUS: APPROVED :taco: :tada: :1st_place_medal: 
Hi :wave: @hendridg , 

appr

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**